lib/rototiller/task/params/env_var.rb

```diff
@@ -43,39 +40,35 @@ def initialize(args={}, &block)
         block_given? ? (yield self) : send_hash_keys_as_methods_to_self(args)
 
         raise(ArgumentError, 'A name must be supplied to add_env') unless @name
-        @set_env ||= true
+        @env_value_set_by_us = false
         reset
       end
 
       # The formatted messages about this EnvVar's status to be displayed to the user
       # @return [String] the EnvVar's message, formatted for color and meaningful to the state of the EnvVar
       def message
-        if message_level == :error
-          level_str = 'ERROR:'
-        elsif message_level == :info
-          level_str = 'INFO:'
-        elsif message_level == :warning
-          level_str = 'WARNING:'
-        end
-        message_prepend = "#{level_str} The environment variable: '#{@name}'"
-        if get_message_type == MESSAGE_TYPES[:default_noexist]
-          return yellow_text("#{message_prepend} is not set. Proceeding with default value: '#{@default}': #{@user_message}")
-        end
-        if get_message_type == MESSAGE_TYPES[:not_required]
-          return yellow_text("#{message_prepend} is not set, but is not required. Proceeding with no flag: #{@user_message}")
-        end
-        if get_message_type == MESSAGE_TYPES[:exist]
-          return green_text("#{message_prepend} was found with value: '#{ENV[@name]}': #{@user_message}")
-        end
-        if get_message_type == MESSAGE_TYPES[:nodefault_noexist]
-          return red_text("#{message_prepend} is required: #{@user_message}")
+        message = String.new
+
+        if env_status    == STATUS[:nodefault_noexist]
+          message << red_text('ERROR: env_var not set and no default provided: ')
+          message << "'#{@name}': '#{@message}'\n"
+        elsif env_status == STATUS[:nodefault_exist]
+          message << yellow_text('INFO: env_var using system value, no default provided: ')
```

Comment:
Will env_var make sense to a user who is consuming the rake task?  This user may not even know that this rakefile implements rototiller.